src/utils/layer-manager-utils.js

```diff
@@ -209,16 +209,16 @@ export const createLayer = (layerConfig) => {
   const Layer = getLayer();
 
   let newLayer;
-  if(portalId){
-    newLayer = new FeatureLayer({
-      portalItem: {
-        id: portalId
-      },
-      title: slug,
-      id: slug,
-      outFields: ['*'],
-    })
-  }else {
+  // if(portalId){
```

Comment:
It would be better to remove this code if we are not going to use it

Reply:
This code will come back once data is fixed. Arcgis seems to have a tough time displaying the color and styles of feature layers unless we used a portal id to load the layer.